src/SFA.DAS.FindEpao.Web/Models/EpaoListItemViewModel.cs

```diff
@@ -26,6 +28,8 @@ public class EpaoListItemViewModel : EpaoWithLocation
         public string Postcode { get; }
         public string Address => FormatAddress();
         public DateTime EffectiveFrom { get; set; }
+        public string[] StandardVersions {get; set;}
+        public string Versions { get; set; }
```

Comment:
The view model shouldn't need both Versions and StandardVersions? The Api returns the list of standards so they could be joined in the constructor or iterate over the list on the view. Happy to disucss

Reply:
Removed Versions & added extension method to perform the array concatenation in the view